### DISCUSSION TITLE
What should Screenwriter do when you generate a new sequence, but already have generated strips in the sequence?

### DISCUSSION THREAD

**Discussion by tin2tin** [tin2tin/Blender_Screenwriter#39] [General]
What should Screenwriter do when you generate a new sequence, but already have generated strips in the sequence?

Should it delete the similar named scenes and objects, before generating the new, or should user data be sacred, and they should not be touched. So users will have to clean up themselves afterwards?

**Comment by tin2tin** [comment]
Maybe a "retime" function could check if the scene/asset/strips exists, and if it does, then only update the properties? But it would also have to check for existing assets if they're still in those scenes. Or when generating some data-base with names for all generated things is saved, and when deleting/removing the data-base will be used as reference. 

Hmm... it's a lot of trouble... an alternative would be a "delete/remove" function to remove all which could be inserted with the current script - but if a line is changed in a dialogue it'll not be deleted. But this may be the easiest way to solve this.

**Reply by b4zz4** [reply]
I could save a list of the scenes generated, erase them and re-generate the new ones.
Without altering our aggregates, but if you destroy our retouchings.
